client/src/partials/patients/record/patient_record.js

```diff
@@ -0,0 +1,25 @@
+angular.module('bhima.controllers')
+.controller('PatientRecordController', PatientRecordController);
+
+PatientRecordController.$inject = ['$stateParams', 'PatientService', 'NotifyService', 'moment'];
+
+function PatientRecordController($stateParams, Patients, Notify, moment) {
+  var vm = this;
+  var patientID = $stateParams.patientID;
+  
+  vm.loading = true;
+  /** @fixme if no uuid is provided this will download all the patients through the base url '/' */
+  Patients.read(patientID)
+    .then(function (result) {
+      vm.patient = result;
+      vm.loading = false;
+      
+      /** @todo move to service or mysql query */
+      vm.patient.name = [vm.patient.first_name, vm.patient.middle_name, vm.patient.last_name].join(' ');
+      vm.patient.age = moment().diff(vm.patient.dob, 'years');
```

Comment:
It could be added as a util method - as it's just one line I've not come across any problems so far, although it's a little bit abstract.